kore/src/main/scala/org/kframework/parser/kore/parser/KoreToK.scala

```diff
@@ -82,7 +82,7 @@ class KoreToK (headToLabel_ : java.util.Properties, sortAtt : Map[k.Sort, Att],
     case kore.Floor(s, rs, p) =>
       throw new TranslationError("Floor patterns currently unsupported")
     case kore.Equals(s1, s2, p, q) =>
-      throw new TranslationError("Equals patterns currently unsupported")
+      KORE.KApply(KLabels.EQUALS_K, apply(p), apply(q))
```

Comment:
I'm not sure this is the way we want to go... I know we've done this in the past, but it might be better to add an `=` operator to kast.k since ==K returns a Bool which isn't exactly the same...